Paymetheus.Decred/Wallet/WalletSeed.cs

```diff
@@ -29,7 +30,18 @@ public static byte[] DecodeAndValidateUserInput(string userInput, PgpWordList pg
                 throw new ArgumentNullException(nameof(pgpWordList));
 
             var decodedInput = DecodeUserInput(userInput, pgpWordList);
-            if (decodedInput.Length != SeedLength)
+            if (decodedInput.Length == (SeedLength + 1))
+            {
+                // deal with "checksum"
+                byte[] d = new byte[SeedLength];
+                Array.Copy(decodedInput, 0, d, 0, SeedLength);
+                var digest = DoubleSha256(d);
+                if (decodedInput[SeedLength] != digest[0])
+                {
+                    throw new Exception($"Invalid checksum, please reenter word list");
```

Comment:
Can use `""` instead of `$""` and don't assume that this was user input, it could come from elsewhere.  Just describe the error.